sklearn/tests/test_discriminant_analysis.py

```diff
@@ -223,6 +225,58 @@ def test_lda_scaling():
                      'using covariance: %s' % solver)
 
 
+def test_lda_store_covariance():
+    # Test for slover 'lsqr' and 'eigen'
+    for solver in ('lsqr', 'eigen'):
+        clf = LinearDiscriminantAnalysis(solver=solver)
+        # The deprecated test will be asserted once the deprecation
```

Comment:
Not sure what this is. But I think the point with these other solvers is to test that `covariance_` is present regardless of `store_covariance` value.